src/functions/template-tags/link.php

```diff
@@ -202,13 +202,13 @@ function tribe_get_listview_link( $term = null ) {
 }
 
 /**
- * Link to List View (Past)
+ * Returns a link to the general or category past view.
```

Comment:
```suggestion
 * Returns a url to the general or category past view.
```
There is confusion some times about link = url vs. link = html anchor tag